engine/worker/cmd_export.go

```diff
@@ -21,7 +21,28 @@ import (
 var cmdExport = &cobra.Command{
 	Use:   "export",
 	Short: "worker export <varname> <value>",
-	Run:   exportCmd,
+	Long: `
+Inside a step [script]({{< relref "workflows/pipelines/actions/builtin/script.md" >}}), you can create a build variable with the worker command:
```

Comment:
isn't it ugly to display hugo templates in a terminal - because the 'long' help is displayed when we do `$ cli command --help`

Reply:
Yes, I'll put long url instead of relref